client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -716,6 +764,8 @@ export default connect(
 		isAtomicSite: isSiteAutomatedTransfer( state, purchase.siteId ),
 		isImport: !! getSiteImportEngine( state, purchase.siteId ),
 		precancellationChatAvailable: isPrecancellationChatAvailable( state ),
+		includedDomainPurchase: getIncludedDomainPurchase( state, purchase ),
```

Comment:
This doesn't seem to be used anywhere. Let's remove it.

Reply:
Thanks!